Cargo.toml

```diff
@@ -19,11 +19,11 @@ paillier = { git = "https://github.com/mortendahl/rust-paillier" }
 [dependencies.cryptography-utils]
 git = "https://github.com/KZen-networks/cryptography-utils"
 features =  ["curvesecp256k1"]
-branch = "freeze-mvp-subzero"
+tag = "v0.0.1"
```

Comment:
Is there any change to cryptography utils that are needed? The branch freeze-mvp-subzero is a super set of the tag v0.0.1

Reply:
I don't think that there is a change. I changed it here only because this is the version used in  multi-party-ecdsa and they must match

Reply:
Ok I will take care of it.